### PR TITLE
Adds multiple rows with index

### PR DESCRIPTION
**Pull request by tr-fi**
Adds multiple rows to the worksheet at the specified index and populates it with values

### CODE REVIEW

**Comment by burnash** [comment]
Thank you very much for your contribution. My bad I've missed this PR and added `append_rows()` from #556 and #694. 
However, I think your contribution is could be used as `insert_rows()` method that gspread is currently missing. For that, we would need to change the method name in the PR and the API call.

Let me know if you're interested in changing the PR.

**Comment by garethsb-sony** [comment]
It would be very nice if the `"insertDimension"` and the values were put atomically. I think it would be possible by using `"updateCells"` ?

**Comment by burnash** [comment]
@garethsb-sony very good point. Thank you for the suggestion. 

For the record, here's the link to `updateCells` docs: [UpdateCellsRequest](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/request#updatecellsrequest)

@tr-fi let me know if you need help with reworking the PR.

**Comment by garethsb-sony** [comment]
Thanks, @burnash ! (I assume `insert_row` should be reworked to call the new `insert_rows` as well, like `append_row` in master calls `append_rows`?)

**Comment by burnash** [comment]
@garethsb-sony yes, exactly. I'm also trying to figure out if the scalar versions of these methods are still needed. Maybe it would be a good idea to deprecate the scalar versions and reduce the API surface a bit.

**Comment by tr-fi** [comment]
@burnash You right insert_rows is more reasonable. So I changed the method name to insert_rows.

File: gspread/models.py [hunk 1]
```diff
@@ -1339,6 +1339,43 @@ def insert_row(
 
         return data
 
+        def insert_rows(self, values, index=1, value_input_option='RAW'):
```
Comment (burnash) [inline gspread/models.py:1342]: Thank you for the update. It looks like the indentation of this line is wrong.
Could you please fix this?

File: gspread/models.py [hunk 2]
```diff
@@ -1339,6 +1339,43 @@ def insert_row(
...
+        :param values: List of row lists.
+        """
+
+        body = {
+            "requests": [{
+                "insertDimension": {
+                    "range": {
+                      "sheetId": self.id,
+                      "dimension": "ROWS",
+                      "startIndex": index - 1,
```
Comment (burnash) [inline gspread/models.py:1357]: It looks like there's a problem with insert indices.

Steps to reproduce:

1. I have a sheet with consecutive values:

Screenshot 2020-04-27 at 15 00 55

2. I call 

```python
wks.insert_rows([['r1.1','r1.2'],['r2.1', 'r2.2']], index=3)
```

Where `wks` is my worksheet.

3. I get 

Screenshot 2020-04-27 at 15 01 20

Where the expected result is 1, 2, r1.1, r2.1, 3, 4, etc in the column A. 
It looks like the method overwrites the cell with value "3".

Comment (tr-fi) [reply gspread/models.py:1357]: Sorry I did not test thoroughly. I fixed it but need you test again

Comment (burnash) [reply gspread/models.py:1357]: No problem, thank you for fast response

**Pull request merged**